src/browser/setupWorker/start/createResponseListener.ts

```diff
@@ -15,10 +15,13 @@ export function createResponseListener(context: SetupWorkerInternalContext) {
   ) => {
     const { payload: responseJson } = message
 
-    // Get the Request instance reference stored in the
-    // request listener.
+    /**
+     * Get the Request instance reference stored in the
+     * request listener for intercepted requests
+     * (bypass and passthrough requests will not be present here).
+     */
     const { requestId } = responseJson
-    const request = context.requests.get(requestId)!
+    const request = context.requests.get(requestId)
```

Comment:
The "non null assertion" going is definitely the right call.  it does need to be handled.